projects/services/consensus-service-typescript/src/transaction-pool-service.ts

```diff
@@ -8,17 +8,32 @@ import { TransactionPool } from "orbs-core-library";
 
 export default class TransactionPoolService extends Service {
   private transactionPool: TransactionPool;
+  private pollInterval: NodeJS.Timer;
+  private pollIntervalMs = 2000;
 
   public constructor(transactionPool: TransactionPool, serviceConfig: ServiceConfig) {
     super(serviceConfig);
     this.transactionPool = transactionPool;
   }
 
   async initialize() {
+    this.startPoolSizeMonitor();
   }
 
   async shutdown() {
+    clearInterval(this.pollInterval);
```

Comment:
@netoneko stopPoolSizeMonitor()?

Reply:
I think we'll be good with this in the mean time.